Lib/trace.py

```diff
@@ -286,7 +286,7 @@ def write_results(self, show_missing=True, summary=False, coverdir=None, *,
             print("lines   cov%   module   (path)")
             for m in sorted(sums):
                 n_lines, percent, modulename, filename = sums[m]
-                print("%5d   %3d%%   %s   (%s)" % sums[m])
+                print("%5d   %3.1f%%   %s   (%s)" % sums[m])
```

Comment:
I think you can use new-style ``.%`` formatting and avoid explicit "percent" (and "sums") computation.  (Now "sums" unpacking right above changed line looks useless.)

Reply:
I meant you can instead store n_hits and do:
```py
n_lines, n_hits, modulename, filename = sums[m]
print(f"{n_lines:5d}   {n_hits/n_lines:.1%}   {modulename}   ({filename})")
```

Reply:
It seems that they are the same, but it doesn't matter, I have changed it ;)

Reply:
Lets keep this suggestion.  Maybe other reviewers also think, that it's a nice use case for ``'%'`` format type;)

Reply:
> I meant you can instead store n_hits and do:
> 
> ```python
> n_lines, n_hits, modulename, filename = sums[m]
> print(f"{n_lines:5d}   {n_hits/n_lines:.1%}   {modulename}   ({filename})")
> ```

@rruuaanng, please consider Sergey's excellent suggestion and use the `%` format specifier.

Reply:
The percent variable is already a percentage, and my local test results show that it must be divided by 100, not by n_lines (but I kept the % format character)

my change
```
print(f"{n_lines:5d}   {n_hits/100:.1%}   {modulename}   ({filename})")
```

```
>>> a = 100
>>> print(f'{a:.1%}')
10000.0%
>>> print(f'{a/100:.1%}')
100.0%
```
Forgive me for not being familiar with the syntax, but all I can tell is that it seems to multiply the value by 100 scale.

Reply:
> Forgive me for not being familiar with the syntax

Please not the difference in your example wrt suggestion.

Nothing strange:
```pycon
>>> f'{0.123:.1%}'
'12.3%'
```
It [well documented](https://docs.python.org/3/library/string.html#format-specification-mini-language): "Percentage. Multiplies the number by 100 and displays in fixed ('f') format, followed by a percent sign."

Reply:
Oh, thanks for your guide, I will read it carefully. This is new knowledge for me :)
(this is more beautiful than using the % character directly.)